src/makefile

```diff
@@ -52,7 +52,7 @@ RSRCS=Space.rsrc
 #		naming scheme you need to specify the path to the library
 #		and it's name
 #		library: my_lib.a entry: my_lib.a or path/my_lib.a
-LIBS= be _APP_
+LIBS= "be sdc++ screensaver"
```

Comment:
Well, my bad it's stdc++ not sdc++

Reply:
Even better, use $(STDCPPLIBS) so it'll work under gcc2 and gcc5.

Reply:
Also, remove the quotes "".
"screensaver" isn't needed either.